core/ArchiveProcessor/Loader.php

```diff
@@ -208,11 +218,22 @@ protected function loadArchiveData()
         // this hack was used to check the main function goes to return or continue
         // NOTE: $idArchives will contain the latest DONE_OK/DONE_INVALIDATED archive as well as any partial archives
         // with a ts_archived >= the DONE_OK/DONE_INVALIDATED date.
-        list($idArchives, $visits, $visitsConverted, $isAnyArchiveExists, $tsArchived, $value) = $this->loadExistingArchiveIdFromDb();
+        $archiveInfo = $this->loadExistingArchiveIdFromDb();
+        $idArchives = $archiveInfo['idArchives'];
+        $visits = $archiveInfo['visits'];
+        $visitsConverted = $archiveInfo['visitsConverted'];
+        $tsArchived = $archiveInfo['tsArchived'];
+        $value = $archiveInfo['doneFlagValue'];
```

Comment:
Apart from the fact it was named like this in the code before, is there possibly some other reason why it's the only variable not following the naming pattern?
```suggestion
        $doneFlagValue = $archiveInfo['doneFlagValue'];
```
Or should it be named possibly something else to better express what value it holds?